engine/server/engine/centralized_logs/client_implementations/persistent_volume/persistent_volume_logs_database_client_test.go

```diff
@@ -154,6 +157,56 @@ func TestStreamUserServiceLogsPerWeek_WithFilters(t *testing.T) {
 	require.NoError(t, testEvaluationErr)
 }
 
+func TestStreamUserServiceLogsPerHour_WithFilters(t *testing.T) {
```

Comment:
Again, nice job with these tests! I know that testing something as complex as this can feel like a drag, but it makes the code sooo much safer 👍